en/peripherals/dshot.md

```diff
@@ -18,17 +19,17 @@ DShot ESCs are connected and wired the same way as [PWM ESCs](pwm_escs_and_servo
 
 If using a Pixhawk flight controller that only has a MAIN port, connect the pins according to the [airframe reference](../airframes/airframe_reference.md) for your vehicle.
 
-If using a Pixhawk that has ports labeled AUX and MAIN, set [SYS_USE_IO=0](../advanced_config/parameter_reference.md#SYS_USE_IO) and connect your ESCs to the AUX-labeled outputs *as though they were labeled MAIN*.
+If using a Pixhawk that has ports labeled AUX and MAIN, set [SYS_USE_IO=0](../advanced_config/parameter_reference.md#SYS_USE_IO) and connect your ESCs to the AUX-labeled outputs *as though they were labeled MAIN*. 
 
 :::note
 A Pixhawk flight controller that has both FMU and IO will label these ports as AUX and MAIN respectively.
-DShot can only be used on the FMU ports (labeled AUX), which is a problem because ESC/motor outputs are typically assigned to the MAIN port in the [airframe reference](../airframes/airframe_reference.md).
+DShot can only be used on the FMU ports (labeled AUX), which is a problem because ESC/motor outputs are typically assigned to the port labelled `MAIN` in the [airframe reference](../airframes/airframe_reference.md). 
 
-To use DShot you therefore normally set `SYS_USE_IO=0` (which makes the ports labeled AUX behave *as though* they were the ports labeled MAIN), and connect your ESCs to the corresponding AUX-labeled outputs.
+To use DShot you therefore normally set `SYS_USE_IO=0` (which makes the ports labeled AUX behave *as though* they were the ports labeled MAIN), and connect your ESCs to the corresponding AUX-labeled outputs. 
 
-Any outputs that would normally be assigned to AUX ports in the [airframe reference](../airframes/airframe_reference.md) are no longer available.
+Any outputs that would normally be assigned to AUX ports in the [airframe reference](../airframes/airframe_reference.md) are no longer available. 
 
-Developers might alternatively modify the [airframe AUX mixer](../dev_airframes/adding_a_new_frame.md#mixer-file) so that the multirotor outputs are on the AUX port rather than MAIN.
+Developers might alternatively modify the [Actuator Configuration](../config/actuators.md) so that motor outputs are on the AUX port rather than MAIN.
 :::
```

Comment:
This whole page needs a check, but in particular this bit. 

Do we still need to talk about disabling the IO? Can you still just use the first 4 AUX ports?
I was thinking we can just say use the actuator config, it will allow you to assign the first 4 AUX ports, otherwise you're stuck?

Further, airframe no longer shows MAIN/AUX in most cases, so we should just refer to assigning outputs right?

Reply:
I would remove `SYS_USE_IO` completely. You can now just re-assing motors on the AUX outputs.

> I was thinking we can just say use the actuator config, it will allow you to assign the first 4 AUX ports, otherwise you're stuck?

Yes, some boards allow more. The UI will show it correctly.

> Further, airframe no longer shows MAIN/AUX in most cases, so we should just refer to assigning outputs/motors using the indicated output function right?

Yes